### PR TITLE
Features/450448 error dialog

### PR DESCRIPTION
**Pull request by molochiy**
Issue 450448 Create error dialog UI

### CODE REVIEW

File: src/app/ui/errordialog.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+ErrorDialog::~ErrorDialog()
+{
+    delete ui;
+    /*!
+     * \brief connect
+     * Connect performs slot on_pushButtonOk_clicked() after pushing on button Ok
+     */
+    connect(pushButtonOk,SIGNAL(clicked(bool)),this,SLOT(on_pushButtonOk_clicked()));
```
Comment (alexchmykhalo) [inline src/app/ui/errordialog.cpp]: Your slot is named like you using auto connection feature. But you are explicitly make connection. Why?

File: src/app/ui/errordialog.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+void ErrorDialog::on_pushButtonOk_clicked()
```
Comment (alexchmykhalo) [inline src/app/ui/errordialog.cpp]: This slot is not needed. See QDialog::accept(), QDialog::reject().  Always use help if you nit sure how to use certain Qt class!

File: src/app/ui/errordialog.h [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+class ErrorDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit ErrorDialog(QWidget *parent = 0);
+    ~ErrorDialog();
+
+private slots:
+    /*!
```
Comment (alexchmykhalo) [inline src/app/ui/errordialog.h:19]: Please, write doxygen comments in cpp file when possible.

**Comment by alexchmykhalo** [comment]
Please, fix the following issues and make pull request again.

**Comment by alexchmykhalo** [comment]
- connect QDialogButtonBox::accepted() to accept() slot
- UI not proportional to mockup
- button box should occupy all width at the bottom of dialog
- no error icon (in designer)
- use text edit for long description

File: src/app/ui/errordialog.cpp [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+#include "ui_errordialog.h"
+
+ErrorDialog::ErrorDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::ErrorDialog)
+{
+    ui->setupUi(this);
+
+    ui->labelBriefErrorText->setWordWrap(true);        
+    ui->plainTextEditFullErrorText-setStyleSheet("border: 2px solid black");
```
Comment (Houd1ny) [inline src/app/ui/errordialog.cpp]: compile error

**Pull request merged**